openapi.yml

```diff
@@ -0,0 +1,670 @@
+openapi: 3.0.0
```

Comment:
This file is a copy/paste of the one I have currently in Profile.

Reply:
Is there anyway to get this from Profile without relying on c/p?

Reply:
We may, but it would involve accessing the repo with a PAT to read the file.
The problem with this solution is we don't want to do that, because we want Connect to work with the client we have implemented because if we don't, what would happen if the file change in Profile?

Reply:
> we want Account to work with the client we have implemented
Good point. I guess we will need to be extra vigilant with CHANGELOG releases from `Connect.Profile`, and manually update this file.

Reply:
Absolutely

Reply:
And be extra careful with the spec versioning, i.e. `version: 1.0.0`

Reply:
Yup, but I didn't start with the changelog / versioning yet.
I never changed that part since I work on profile and I think we should change it there, but there's also small changes to make inside the file for it to be truly "clean".
I think we may consider this PR as a first step, but we will have to use the new version (I have not written yet) asap

Reply:
Can you change the version to `0.1.0` then? :)

Reply:
Not without a PR on Profile, but I add a task for that and you'll get the new version with the next Profile's changes if you're ok with that